utils/releaser/src/linux/desktop.rs

```diff
@@ -150,13 +149,13 @@ pkgbase = lockbook-desktop
 	pkgver = {version}
 	pkgrel = 1
 	url = https://github.com/lockbook/lockbook
-	arch = x86_64
-	arch = i686
+	arch = any
```

Comment:
Since you made this `any`, you should also add it to the PKGBUILD: `arch=('any')`